sources/osg/Polytope.js

```diff
@@ -335,12 +332,65 @@ Polytope.prototype = MACROUTILS.objectInherit( Object.prototype, {
         for ( var i = 0; i < this._planeList.length; ++i ) {
             if ( this._resultMask & selectorMask ) {
                 Plane.transformProvidingInverse( this._planeList[ i ], matrix );
-                selectorMask <<= 1;
             }
+            selectorMask <<= 1;
         }
-    }
+    },
 
+    /** Check whether any part of a triangle is contained within the polytope.*/
+    contains: function ( v0, v1, v2 ) {
+        if ( !this._maskStack[ this._maskStack.length - 1 ] ) return;
+
+        // Note check whether it could be worth to do calculations in floats.
+        var src = [];
+        var dest = [];
+
+        src.push( v0 );
+        src.push( v1 );
```

Comment:
fine by me because the function won't be used in sketchfab but all the pushes and dynamic allocation (slice in the foor loop) would have been problematic